pkg/queue/sharedmain/main.go

```diff
@@ -424,8 +425,9 @@ func setupMetricsExporter(ctx context.Context, logger *zap.SugaredLogger, backen
 		Component:      "revision",
 		PrometheusPort: networking.UserQueueMetricsPort,
 		ConfigMap: map[string]string{
-			metrics.BackendDestinationKey: backend,
-			"metrics.opencensus-address":  collectorAddress,
+			metrics.BackendDestinationKey:      backend,
+			"metrics.opencensus-address":       collectorAddress,
```

Comment:
We need to make constants public at the knative/pkg repo to avoid hardcoding values. Will fix in another PR.